extensions/spark/kyuubi-spark-connector-tpch/src/test/resources/log4j2-test.properties

```diff
@@ -0,0 +1,40 @@
+#
```

Comment:
Use log4j.properties rather than log4j2, @wForget is working on it in TPCDS module